### PR TITLE
Update get_ground_diffuse description

### PR DESCRIPTION
**Pull request by AdamRJensen**
- [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

There exist several methods in the literature for estimating ground-reflected diffuse irradiance. Thus, the pvlib method should clearly state that it is based on isotropic ground reflections (which is the most common method)

### CODE REVIEW

**Review by cwhanse — COMMENTED** [review]
I approve of the clarification.

Fussing about words: should it be "The ground is assumed to be isotropic and reflections are Lambertian".

File: pvlib/irradiance.py [hunk 1]
```diff
@@ -542,7 +542,8 @@ def poa_components(aoi, dni, poa_sky_diffuse, poa_ground_diffuse):
 def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
     '''
     Estimate diffuse irradiance from ground reflections given
-    irradiance, albedo, and surface tilt.
+    irradiance, albedo, and surface tilt. The ground is assumed to
+    be isotropic and reflections are Lambertian.
```
Comment (adriesse) [inline pvlib/irradiance.py:546]:
```suggestion
    be horizontal and flat, and reflections are Lambertian.
```
I don't think the word isotropic is appropriate here.

Comment (cwhanse) [reply pvlib/irradiance.py:546]: why not?

Comment (adriesse) [reply pvlib/irradiance.py:546]: If you want to use the word isotropic I would suggest something more like: the surface is Lambertian and the reflected radiance is isotropic.

Comment (cwhanse) [reply pvlib/irradiance.py:546]: I can live with that.

**Comment by jules-ch** [comment]
+1 on this clarification

**Comment by cwhanse** [comment]
@kandersolar @AdamRJensen please take a look, moved this forward to v0.10.4

**Comment by AdamRJensen** [comment]
@cwhanse Do you have a source that explains the difference between isotropic and Lambertian?

**Comment by cwhanse** [comment]
> @cwhanse Do you have a source that explains the difference between isotropic and Lambertian?

If a surface is Lambertian, then reflected radiance is isotropic. https://en.wikipedia.org/wiki/Lambertian_reflectance

**Comment by cwhanse** [comment]
A surface (or a material) can also be isotropic, which means that its properties don't depend on position or direction.

**Comment by AdamRJensen** [comment]
This PR looks good to me!

**Review by kandersolar — APPROVED** [review]

**Comment by mikofski** [comment]
Maybe irrelevant but the opposite of Lambertian is “specular” like a mirror, a specular surface only reflects light in a single direction: https://en.m.wikipedia.org/wiki/Specular_reflection. A Lambertian surface reflects light in more than one direction

**Pull request merged**